### ISSUE TITLE
COVID-19 dashboards powered by TARGIT

### ISSUE THREAD

**Issue by ODrying** [CSSEGISandData/COVID-19#230]
So far I have created three dashboards:

- Map, KPIs and graphs for Accumulated Confirmed
- Map, KPIs and graphs for New Confirmed
- Country list and graphs for countries comparison

More to be added. Let me know what you think.
![2020-03-04_15-41-20](https://user-images.githubusercontent.com/61792255/75890435-a7815900-5e2e-11ea-91ed-c2df70582369.png)

**Comment by nysky1** [comment]
This is better than the Johns Hopkins UI. Can you begin to update the dataset daily?

**Comment by nysky1** [comment]
If you will commit to updating the data daily, I'll promote the usage with the press.

**Comment by ODrying** [comment]
@nysky1 The dataset is updated automatically. It is pulling data from CSSE at JHU, as mentioned in the upper left corner of the dashboards. As long as CSSE keep updating the GitHub CSVs, the dashboards will update accordingly. Thank you.

**Comment by nysky1** [comment]
Shouldn't your Accum. Case # be the same as Johns Hopkins for today? Or is JH supplementing with non-csv data?

**Comment by ODrying** [comment]
@nysky1 No. The Johns Hopkins map is pulling on near real time data. They have their sources. :-) The CSV files are updated less frequently - as infrequent as once per day. I think it is fair enough that John Hopkins are more up-to-date, keeping in mind the effort needed of pulling together these data and subsequently making them publicly available to data nerds like me.

**Comment by ODrying** [comment]
E.g. right now, the CSV files has a time stamp 19 hours ago.

**Comment by nysky1** [comment]
Thanks for the clarification. Your filtering is tremendously more insightful as we look to the epicenter for insight.
